test/e2e/config/vsphere.yaml

```diff
@@ -271,6 +275,9 @@ variables:
   EXP_NODE_ANTI_AFFINITY: "true"
   CAPI_DIAGNOSTICS_ADDRESS: ":8080"
   CAPI_INSECURE_DIAGNOSTICS: "true"
+  # Required to be set to install capv-supervisor <= v1.10, but getting created.
```

Comment:
Not sure I understand what "but getting created" means

Reply:
It is required to install CAPV in supervisor mode to a management cluster for versions <= v1.10.

Reply:
Yup that makes sense. I just can't parse the last part of the sentence :D

Reply:
Fine to address in a follow-up